project1/client/client_main.cpp

```diff
@@ -0,0 +1,28 @@
+/**
+ * @brief Entry point for the FTP client.
+ */
+
+
+#include "client.h"
+#include "input_parser/input_parser.h"
+#define PORT 8080
+namespace client {
```

Comment:
`main` can't be in a namespace for this to work. I think the only reason this compiles at all is because you're not actually making an executable. (See my comment above.)